TwitchLib.EventSub.Core/SubscriptionTypes/Channel/HypeTrainProgressV2.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using TwitchLib.EventSub.Core.Models.HypeTrain;
+
+namespace TwitchLib.EventSub.Core.SubscriptionTypes.Channel;
+
+/// <summary>
+/// HypeTrain Progress subscription type model
+/// <para>Description:</para>
+/// <para>A Hype Train makes progress on the specified channel.</para>
+/// </summary>
+public sealed class HypeTrainProgressV2
+{
+    /// <summary>
+    /// The event ID.
+    /// </summary>
+    public string Id { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The requested broadcaster ID.
+    /// </summary>
+    public string BroadcasterUserId { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The requested broadcaster login.
+    /// </summary>
+    public string BroadcasterUserLogin { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The requested broadcaster display name.
+    /// </summary>
+    public string BroadcasterUserName { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Total points contributed to the Hype Train.
+    /// </summary>
+    public int Total { get; set; }
+
+    /// <summary>
+    /// The number of points contributed to the Hype Train at the current level.
+    /// </summary>
+    public int Progress { get; set; }
+
+    /// <summary>
+    /// The number of points required to reach the next level.
+    /// </summary>
+    public int Goal { get; set; }
+
+    /// <summary>
+    /// The contributors with the most points contributed.
+    /// </summary>
+    public HypeTrainContribution[] TopContributions { get; set; } = [];
+
+    /// <summary>
+    /// The current level of the Hype Train.
+    /// </summary>
+    public int Level { get; set; }
+
+    /// <summary>
+    /// The all-time high level this type of Hype Train has reached for this broadcaster.
+    /// </summary>
+    public int AllTimeHighLevel { get; set; }
```

Comment:
AlltimeHigh not in docs. 
https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelhype_trainprogress-v2

Reply:
Fixed